samples/dotnet/kernel-syntax-examples/Example12_SequentialPlanner.cs

```diff
@@ -130,12 +131,21 @@ private static async Task BookSamplesAsync()
         await ExecutePlanAsync(kernel, originalPlan);
     }
 
-    private static string PlanToString(Plan originalPlan)
+    private static string PlanToString(Plan originalPlan, string indent = " ")
     {
-        return $"Goal: {originalPlan.Description}\n\nSteps:\n" + string.Join("\n", originalPlan.Steps.Select(
-            s =>
-                $"- {s.SkillName}.{s.Name} {string.Join(" ", s.NamedParameters.Select(p => $"{p.Key}='{p.Value}'"))}{" => " + string.Join(" ", s.NamedOutputs.Where(s => s.Key.ToUpper(System.Globalization.CultureInfo.CurrentCulture) != "INPUT").Select(p => $"{p.Key}"))}"
-        ));
+        var sb = new StringBuilder($"{indent}Goal: {originalPlan.Description}\n\n{indent}Steps:\n");
+
+        foreach (var step in originalPlan.Steps)
+        {
+            sb.Append(step.Steps.Count switch
+            {
+                0 =>
+                    $"{indent}{indent}- {string.Join(".", step.SkillName, step.Name)} {string.Join(" ", step.NamedParameters.Select(p => $"{p.Key}='{p.Value}'"))}{(step.NamedOutputs.Where(s => s.Key.ToUpper(System.Globalization.CultureInfo.CurrentCulture) != "INPUT").Select(p => $"{p.Key}").FirstOrDefault() is var namedOutputs ? $" => {namedOutputs}" : "")}",
+                _ => PlanToString(step, indent + indent)
+            });
+        }
+
```

Comment:
note:

Reply:
If I use both ActionPlanner and SequentialPlanner I must do the following. This is because ActionPlanner is a skill as well.

> var planner = new SequentialPlanner(kernel, new SequentialPlannerConfig() { ExcludedSkills = new[] { "this" } });

Reply:
is there a way not to load the actionplanner skill in the kernel?

Reply:
Yes. But if you try to use both, this will happen.